### ISSUE TITLE
Add devel rules for all gnome ports

### ISSUE THREAD

**Issue by AMDmi3** [repology/repology-rules#262]
As per [report for gedit-plugins](https://repology.org/project/gedit-plugins/report) we need anticipatory devel rules (for odd versions) for all gnome ports, to not make people submit reports on every odd version released. The list of remaining gnome ports may likely be discovered with a single query:

```sql
SELECT
    effname,
    max(version::versiontext) FILTER (WHERE versionclass != 10) AS version  -- exclude rolling
FROM packages
GROUP BY effname
HAVING
    count(*) FILTER(WHERE version LIKE '3.32.%') > 1 AND
    count(*) FILTER(WHERE (flags & 2)::boolean) = 0
ORDER BY effname;
```

E.g. get projects which have version looking like gnome and no devel flags (the latter is not quite accurate, but it produces false positives rather than false negatives, so it should be OK).

The results are currently:
```
                effname                | version
---------------------------------------+----------
 devhelp                               | 3.32.0
 five-or-more                          | 3.32.0
 gdm-plymouth                          | 3.32.0
 gdm-prime                             | 3.32.0
 gedit-plugins                         | 3.33.90
 gitg                                  | 3.32.1
 gnome-applets                         | 3.32.0
 gnome-authenticator                   | 3.32.2
 gnome-color-manager                   | 3.32.0
 gnome-devel-docs                      | 3.32.1
 gnome-games-app                       | 3.32.0
 gnome-latex                           | 3.32.0
 gnome-multi-writer                    | 3.32.1
 gnome-notes                           | 3.32.2
 gnome-packagekit                      | 3.32.0
 gnome-passwordsafe                    | 3.32.0
 gnome-power-manager                   | 3.32.0
 gnome-shell-extension-background-logo | 3.32.0
 gnome-sushi                           | 3.32.0
 gnome-terminal-fedora                 | 3.32.2
 gpaste                                | 3.32.0
 gtkhtml3                              | 3.32.2
 libwnck3                              | 3.32.0
 lightsoff                             | 3.32.0
 mailnag-gnome-shell                   | 3.32.1
 mingw:adwaita-icon-theme              | 3.32.0
 mousetweaks                           | 3.32.0
 nautilus-typeahead                    | 3.32.3
 python:braintree                      | 3.56.0
 python:gobject3-common                | 3.32.2
 r:rmat                                | 3.32.0
 swell-foop                            | 3.32.0
 tali                                  | 3.32.1
 zenity                                | 3.32.0
 zukitwo                               | 20170209
```

Notably, it detects gedit-plugins.
